src/.nuget/NEventStore.Persistence.RavenDB.nuspec

```diff
@@ -14,11 +14,12 @@
     <tags>events, event sourcing, cqrs, storage, persistence, database, ravendb</tags>
     <dependencies>
       <dependency id="NEventStore" version="5.2" />
-      <dependency id="RavenDB.Client" version="[2.5, 3.0)" />
+      <dependency id="RavenDB.Client" version="[3.0.3660, 4.0)" />
+      <dependency id="RavenDB.Database" version="[3.0.3660, 4.0)" />
```

Comment:
just found that [NEventStore.Persistence.RavenDB in nuget](https://www.nuget.org/packages/NEventStore.Persistence.RavenDB) still showing dependency `[2.5, 3.0)`, any ideas why?